tests/pages/services/ServiceFormModal-cy.js

```diff
@@ -1648,5 +1648,75 @@ describe("Service Form Modal", function() {
         });
       });
     });
+
+    context("Multi-container (pod)", function() {
+      beforeEach(function() {
+        cy
+          .get(".menu-tabbed-item-label")
+          .eq(0)
+          .click()
+          .get(".menu-tabbed-view h2")
+          .contains("Service");
+      });
+
+      it("Should add new container", function() {
+        cy.get(".menu-tabbed-view .button.button-primary-link").eq(3).click();
```

Comment:
Am I right that here we test if clicking on the button doesn't throw an Exception? Description says "Should add new container" but we don't check that container is added anywhere.